### PR TITLE
zh_cn localization++

### PR DESCRIPTION
**Pull request by ZZZank**
This PR should include translations for all entries in `/resources/***/lang/` , except for quests
## Summary
It includes Chinese localization for most entries in `/resources/***/lang/` , including: 
- autotooltips
- command messages
- corrections on `Elektron`
- custom Mek infusing type
- custom NuclearCraft parts
- custom Thaumcraft aspects
- tips

## Notes
Custom Thaumcraft entries are not localized yet, because there have been some noticible changes since I started this PR, I think it would be better to start a new, rebased fork. 
NuclearCraft lang file includes entries for configs. They are not localized, and should not be localized. 
localization for `Elektron` is based on info from https://encyclopedia.thefreedictionary.com/Elektron+(alloy) .

### CODE REVIEW

**Comment by Krutoy242** [comment]
Sounds awesome! I think this would be great for Chinese community.

File: resources/tips/lang/en_us.lang
Comment (Krutoy242) [inline resources/tips/lang/en_us.lang:1]: Please make another PR with this changes.

**Comment by Krutoy242** [comment]
Notes:

- You dont need to translate `resources/nuclearcraft/lang/zh_cn.lang` until lines `# New Turbine Parts`
  > This is a temporary fix for NC configuration file since mod dev forget to add localizations. Config locals should be on EN.
- Please keep fixes of EN langs (like ones you did in `resources/tips/lang/en_us.lang`) in separate PR since it would be hard to track changes later. One feature - one PR.

File: resources/tips/lang/en_us.lang
Comment (ZZZank) [reply resources/tips/lang/en_us.lang:1]: Sure, #279 , I forgot it

**Comment by ZZZank** [comment]
The localization is mostly done now, and there's only one problems that really confuses me: What does 'Elektron' mean? Or to be exact, what does 'Elektron' mean in these 3 entries?
```
tile.nuclearcraft.turbine_rotor_blade_elektron90.name=Unwieldy Elektron90 Alloy Turbine Rotor Blade
tile.nuclearcraft.turbine_rotor_blade_elektron192.name=Unwieldy Elektron192 Alloy Turbine Rotor Blade
tile.nuclearcraft.turbine_rotor_blade_elektron231.name=Unwieldy Elektron231 Alloy Turbine Rotor Blade
```
The last time I met with this word is when I was translating `Elektron60 Alloy` in `resources\contenttweaker\zh_cn.lang` . I tried search this word, and online dictionary tells me that it refers to Magnesium-aluminum Alloy. 
Obviously it's not, because the recipe defined in `scripts\mods\nuclearcraft\NTP\other recipes.zs` says it's Magnesium26 + Cobalt60. So finally I use "Magnesium-Cobalt60 Alloy" as its Chinese translation. 

But the problem is, now I know that the form `Elektron{num}` appears many times, which might refers to a series of alloys that I'm not familiar with. So I think I needs some help now. @Krutoy242

**Comment by Krutoy242** [comment]
I don't know what Elektron means too! I'm gonna try and ask `New Turbine Parts` developer about it.

**Comment by Krutoy242** [comment]
Adden dev said that `Electron` is a trade mark https://en.m.wikipedia.org/wiki/Elektron_(alloy)

**Comment by ZZZank** [comment]
> Addon dev said that `Electron` is a trade mark https://en.m.wikipedia.org/wiki/Elektron_(alloy)

I can't access wikipedia, so I use informations from https://encyclopedia.thefreedictionary.com/Elektron+(alloy) to guide my translation, are these infomations the same as those from wikipedia?

**Comment by Krutoy242** [comment]
Yep, this is mostly copy-paste of Wikipedia entry.

**Comment by ZZZank** [comment]
Review is done(no problem found), this PR good to go now!
